Saas/Get-BcNotificationRecipients.ps1

```diff
@@ -0,0 +1,45 @@
+<#
+ .Synopsis
+  Function for retrieving the notification recipients configured for a Business Central tenant.
+ .Description
+  Function for retrieving the notification recipients configured for a Business Central tenant.
+  Returns Id, Email and Name configured for each notification recipient.
+  Wrapper for https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/administration/administration-center-api_notifications#get-notification-recipients
+ .Parameter bcAuthContext
+  Authorization Context created by New-BcAuthContext.
+ .Parameter apiVersion
+  API version. Default is v2.15.
+ .Example
+  $bcauthContext = New-BcAuthContext -includeDeviceLogin
+  Get-BcNotificationRecipients -bcAuthContext $bcauthContext
+#>
+
+function Get-BcNotificationRecipients {
+    Param(
+        [Parameter(Mandatory = $true)]
+        [Hashtable] $bcAuthContext,
+        [string] $apiVersion = "v2.6"
+    )
+
+    $telemetryScope = InitTelemetryScope -name $MyInvocation.InvocationName -parameterValues $PSBoundParameters -includeParameters @()
+    try {
+
+        $bcAuthContext = Renew-BcAuthContext -bcAuthContext $bcAuthContext
```

Comment:
Please use the same mechanism as used in the other saas functions:

```
        $bcAuthContext, $headers, $endPointURL = Create-SaasUrl -bcAuthContext $authContext -endPoint "usedstorage" -environment $environment -applicationFamily $applicationFamily -apiVersion $apiVersion
```

to get headers, bcauthcontext and url.